app/src/screens/Splash.tsx

```diff
@@ -205,30 +239,54 @@ const Splash: React.FC = () => {
         newAgent.registerOutboundTransport(wsTransport)
         newAgent.registerOutboundTransport(httpTransport)
 
+        setProgress(t('Init.InitializingAgent'), 55)
```

Comment:
Instead of a hardcoded number can we just do some division based on number of steps?

Reply:
Updated